includes/cart/actions.php

```diff
@@ -101,7 +101,7 @@ function edd_process_remove_from_cart( $data ) {
 function edd_process_remove_fee_from_cart( $data ) {
 	$fee = sanitize_text_field( $data['fee'] );
 	EDD()->fees->remove_fee( $fee );
-	wp_redirect( remove_query_arg( array( 'edd_action', 'fee' ) ) ); edd_die();
+	wp_redirect( remove_query_arg( array( 'edd_action', 'fee', 'nocache' ) ) ); edd_die();
```

Comment:
B/C this is the redirect `after` the removal happens. we've already removed, so get that off otherwise it removes the item then redirects to the cart with only the `nocache` query string, we've already removed the `fee` and `action` items after it's removed.